MockKitUI/MockCGMManager+UI.swift

```diff
@@ -14,6 +14,8 @@ import MockKit
 
 
 extension MockCGMManager: CGMManagerUI {
+    public var smallImage: UIImage? { UIImage(systemName: "dial") }
```

Comment:
Why a dial?

Reply:
Oh, I just picked something, just to make it show _something_.  And I didn't want to add an image, so I picked a system image that looked somewhat relevant.  If you feel strongly I can choose something else.

Reply:
Is leaving it blank not an option? I think the simulators should either not have an image, or have something at least somewhat communicative.